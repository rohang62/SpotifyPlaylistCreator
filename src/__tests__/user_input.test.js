import React from "react";
import renderer from "react-test-renderer";
import UserInput from "./../user_input";

test("renders user input screen", () => {
	let state = { username : '', password : '' };
	const Tree = renderer.create(<UserInput state = {state}/>).toJSON();
	expect(Tree).toMatchSnapshot();
});
