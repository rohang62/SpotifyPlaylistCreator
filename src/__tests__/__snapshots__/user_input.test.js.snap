// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders user input screen 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="spotify_logo.svg"
    />
    <div>
      <label>
        Name of Playlist to create:
        <input
          onChange={[Function]}
          type="text"
          value=""
        />
      </label>
    </div>
    <div>
      <label>
        Length of Playlist:
        <input
          max="600"
          min="10"
          onChange={[Function]}
          type="range"
          value={100}
        />
        100
         
        minutes
      </label>
    </div>
    <div>
      <button
        className="button"
        onClick={[Function]}
      >
         Go back to Login 
      </button>
      <div
        className="divider"
      />
      <button
        className="button"
        disabled={true}
        onClick={[Function]}
      >
         
        Create Playlist
      </button>
    </div>
  </header>
</div>
`;
